libs/ui-lib/lib/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfiguration.tsx

```diff
@@ -102,18 +98,6 @@ const isManagedNetworkingDisabled = ({
       networkManagementDisabledReason:
         'Network management selection is not supported with dual-stack',
     };
-  } else if (
-    isArmCpu &&
-    !featureSupportLevelData.isFeatureSupported(
-      'ARM64_ARCHITECTURE_WITH_CLUSTER_MANAGED_NETWORKING',
-    )
-  ) {
-    return {
-      isNetworkManagementDisabled: true,
-      networkManagementDisabledReason: featureSupportLevelData.getFeatureDisabledReason(
-        'ARM64_ARCHITECTURE_WITH_CLUSTER_MANAGED_NETWORKING',
-      ),
-    };
   } else if (
     !!openshiftVersion &&
```

Comment:
I think `openshiftVersion` is not actually needed, we can also clean it up.

Reply:
Only the `!!openshiftVersion` as it was there to make Typescript happy, but if it's not being used in the function call, we don't need it anymore.
It reduces the hook dependencies etc